aliases/available/clipboard.aliases.bash

```diff
@@ -2,11 +2,14 @@ cite 'about-alias'
 about-alias 'pbcopy and pbpaste shortcuts to linux'
 
 case $OSTYPE in
-  linux*)
-    XCLIP=$(command -v xclip)
-    [[ $XCLIP ]] && \
-    alias pbcopy="$XCLIP -selection clipboard" && \
-    alias pbpaste="$XCLIP -selection clipboard -o"
+    linux*)
```

Comment:
I think you can remove the `$OSTYPE` filter here and just use `command_exists xclip` check instead. you can also get rid of `$XCLIP`

Reply:
@NoahGorny Done